ledger/src/blockstore.rs

```diff
@@ -7756,20 +7744,17 @@ pub mod tests {
             ));
         }
 
-        // Establish a baseline that works
-        coding_shred.set_index(coding_shred.index() + 1);
         assert!(Blockstore::should_insert_coding_shred(
-            &coding_shred,
+            &code_shreds[1],
             max_root
         ));
 
         // Trying to insert value into slot <= than last root should fail
         {
-            let mut coding_shred = coding_shred.clone();
-            coding_shred.set_slot(max_root);
+            let coding_shred = coding_shred.clone();
```

Comment:
No need to clone here right ?

Reply:
Yep it does literally nothing.  Addressed in https://github.com/anza-xyz/agave/pull/5731/commits/a5991dcef834859abc6acfdad0d7fa61a14b5338